src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Link.cs

```diff
@@ -341,6 +353,14 @@ void ProcessManifest (ITaskItem manifestFile)
 			}
 		}
 
+		string GetManifestRulesFile (string manifestDir)
+		{
+			string rulesFile = Path.Combine (manifestDir, "aapt_rules.txt");
```

Comment:
Does anything need to delete this on `Clean`? Or is it in a folder we already delete?

Reply:
These are picked up by https://github.com/xamarin/xamarin-android/pull/7587/files#diff-4fd605fa8c8f1a730daade6811197fcb16dd3640df56117d5e6da65093dde9fcR247